temporian/core/evaluation.py

```diff
@@ -16,34 +16,50 @@
 
 import time
 import sys
-import pathlib
-from typing import Any, Dict, List, Set, Union
+from typing import Any, Dict, List, Set, Tuple, Union
 from collections import defaultdict
 
 from temporian.core.data.node import Node
 from temporian.core.operators import base
-from temporian.implementation.numpy.data.event_set import EventSet
 from temporian.core import processor as processor_lib
-from temporian.implementation.numpy import evaluator as numpy_evaluator
+from temporian.implementation.numpy import evaluation as np_eval
+from temporian.implementation.numpy.data.event_set import EventSet
+
 
-AvailableBackends = Any
-Data = Dict[Node, Union[str, pathlib.Path, EventSet]]
-Query = Union[Node, List[Node], Dict[str, Node]]
-Result = Union[EventSet, List[EventSet], Dict[str, EventSet]]
+EvaluationQuery = Union[Node, List[Node], Dict[str, Node]]
```

Comment:
its used to return a dict with same keys as the input, if the input is a dict. I agree its unclear what they should contain - because its arbitrary, can be anything. will add that to the docstring.

```
If `query` is a dictionary of nodes, the returned object will be a
        dictionary of event sets.
```